features/partners.feature

```diff
@@ -5,11 +5,10 @@ Functionaliteit: Huwelijken en geregistreerd partnerschappen van een persoon raa
   Dit bevat de gegevens over de relatie (huwelijk of partnerschap) plus enkele identificerende eigenschappen van de persoon waarmee het huwelijk of partnerschap is aangegaan.
 
   # RvIG levert personen in de vorm GbaPersoon, waarin alleen onbewerkte GBA-V gegevens worden geleverd
-  # Actuele partners is (huwelijk / partnerschap is niet ontbonden) worden geleverd
-  # Als er een (of meer) actuele pertner is worden geen ontbonden huwelijken/partnerschappen geleverd
-  # Als er alleen ontbonden huwelijken/partnerschappen zijn levert de RvIG alleen de ex-partner met de meest recente datum ontbinding
+  # Dit betekent dat in 'partners' een actuele partner of een ontbonden partner kan bevatten
```

Comment:
kan je het woordje "in" weghalen, zodat het een correcte zin wordt?

Reply:
onbewerkt betekent niet hetzelfde als ongefilterd. De RvIG levert de actuele óf de laatst ontbonden partner

Reply:
Niet bewust verwijderd. Deze 2 zijn tussen wal en schip beland door de merge-issues. Ik had regel 8 opgenomen voor deze uitleg. Alsnog nu de regels 9 en 10 weer opgenomen.